src/commands/leaderboard.py

```
@@ -28,94 +28,187 @@
 
 
+def _get_pick_correct_attr():
+    """Return the attribute on Pick that represents correctness.
+
+    Tries common names so this file doesn't hard-fail if the field name
+    differs slightly across schema versions.
+    """
+    for name in ("correct", "is_correct", "was_correct"):
+        if hasattr(Pick, name):
+            return getattr(Pick, name)
+    raise AttributeError("Pick model has no attribute indicating correctness")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)